resources/default-settings/nginx.default

```diff
@@ -2,7 +2,7 @@ server {
 	listen 80 default_server;
 	listen [::]:80 default_server;
 
-	root /home/pi/RPi-Jukebox-RFID/src/webapp/build;
```

Comment:
Why is this important? nginx was able to access it before, right?!

Reply:
It didn't work in my installation, because nginx could not access the home dir (running with limited 'nginx' user), but I just realized that was caused by my test environment. Raspberry Pi OS allows all users by default access to all home directories. I will remove the commit.